modules/dnn/src/onnx/onnx_importer.cpp

```diff
@@ -398,7 +398,6 @@ void ONNXImporter::populateNet(Net dstNet)
         std::string layer_type = node_proto.op_type();
         layerParams.type = layer_type;
 
-
```

Comment:
Please avoid unrelated changes (extra lines addings or removals)

Reply:
Ah! Sorry. I will remove it.

Reply:
Never mind. Just to make this PR perfect :)